src/sql/src/pure.rs

```diff
@@ -384,7 +385,213 @@ pub async fn purify_create_source(
                 )))),
             })
         }
-        CreateSourceConnection::LoadGenerator { .. } => (),
+        CreateSourceConnection::LoadGenerator {
+            generator,
+            options: _,
+        } => {
+            use mz_storage::types::sources::LoadGenerator;
+
+            let scx = StatementContext::new(None, &*catalog);
+
+            let available_subsources = match generator {
+                mz_sql_parser::ast::LoadGenerator::Auction => {
+                    let mut available_subsources = HashMap::new();
+                    let generator = as_generator(&LoadGenerator::Auction);
+                    for (i, (name, desc)) in generator.views().iter().enumerate() {
+                        let name = FullObjectName {
+                            database: RawDatabaseSpecifier::Name("loadgenerator".to_owned()),
```

Comment:
I have not chased this down whatsoever, but it's convention for namespaces/objects we create to get prefixed with `mz_`; seems like this database name should follow suit. Not sure who ought to manage that, but wanted to flag in case you felt inclined.